src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/BaseObjects.cs

```diff
@@ -291,6 +291,27 @@ public class SchedulePolicyBase : ObjectBase
     {
     }
 
+    /// <summary>
+    /// Base class for azure workload policy settings.
+    /// </summary>
+    public class SettingsBase : ObjectBase
+    {
+        /// <summary>
+        /// Is compression enabled
+        /// </summary>
+        public bool? IsCompression { get; set; }
```

Comment:
Going forward we will have only one of these, resolve it now to avoid breaking changes